seatunnel-connectors-v2/connector-paimon/pom.xml

```diff
@@ -30,7 +30,7 @@
     <name>SeaTunnel : Connectors V2 : Paimon</name>
 
     <properties>
-        <paimon.version>0.4.0-incubating</paimon.version>
+        <paimon.version>0.6.0-incubating</paimon.version>
```

Comment:
Compatibility needs to be tested to see if the new connector can still be used on payment 0.4.0-incubating.

Reply:
Yes, my production environment is paimon 0.6, and I re-typed the connector package to replace the connector of the production environment for testing, which is compatible.

Reply:
At present, there shouldn't be too many changes to Paimon's API. I think the compatibility issue can be temporarily ignored. Upgrading Paimon to the latest version is okay.